### ISSUE TITLE
Solr error when importing ISA archives on AWS

### ISSUE THREAD

**Issue by hackdna** [parklab/refinery-platform#1116]
Commit: 9f9b0dcb

### Steps to reproduce
1. Spin up a Refinery stack on AWS
1. Import an ISA archive (for example: [rfc-test.zip](https://s3.amazonaws.com/data.cloud.refinery-platform.org/data/sample-data/isa-tabs/rfc-test.zip))

### Observed behavior
Data set contents is displayed in the data set view (`data_sets/<UUID>/#/files/browse`). However, the following error is logged in the Celery worker 1 log:
```
2016-05-11 14:37:57 INFO     core.search_indexes:145 prepare() - Successfully prepared 'Test 1: Request for Comments (RFC) Test' for indexing
[2016-05-11 14:37:57,370: ERROR/Worker-4] [Reason: None]
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/><title>Error 500 {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</title></head><body><h2>HTTP ERROR 500</h2><p>Problem accessing /solr/core/update/. Reason:<pre>    {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217)	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</pre></p><hr><i><small>Powered by Jetty://</small></i><hr/></body></html>
Failed to add documents to Solr: [Reason: None]
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/><title>Error 500 {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</title></head><body><h2>HTTP ERROR 500</h2><p>Problem accessing /solr/core/update/. Reason:<pre>    {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217)	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</pre></p><hr><i><small>Powered by Jetty://</small></i><hr/></body></html>
[2016-05-11 14:37:57,371: ERROR/Worker-4] Failed to add documents to Solr: [Reason: None]
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/><title>Error 500 {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</title></head><body><h2>HTTP ERROR 500</h2><p>Problem accessing /solr/core/update/. Reason:<pre>    {msg=SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;,trace=org.apache.solr.common.SolrException: SolrCore &apos;core&apos; is not available due to init failure: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.CoreContainer.getCore(CoreContainer.java:974)	at org.apache.solr.servlet.HttpSolrCall.init(HttpSolrCall.java:250)	at org.apache.solr.servlet.HttpSolrCall.call(HttpSolrCall.java:417)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:214)	at org.apache.solr.servlet.SolrDispatchFilter.doFilter(SolrDispatchFilter.java:179)	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1652)	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:585)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:577)	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:223)	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1127)	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:515)	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:185)	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1061)	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:215)	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:110)	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:97)	at org.eclipse.jetty.server.Server.handle(Server.java:499)	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:310)	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:257)	at org.eclipse.jetty.io.AbstractConnection$2.run(AbstractConnection.java:540)	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)	at java.lang.Thread.run(Thread.java:745)Caused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:820)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:659)	at org.apache.solr.core.CoreContainer.create(CoreContainer.java:723)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:443)	at org.apache.solr.core.CoreContainer$1.call(CoreContainer.java:434)	at java.util.concurrent.FutureTask.run(FutureTask.java:262)	at org.apache.solr.common.util.ExecutorUtil$MDCAwareThreadPoolExecutor$1.run(ExecutorUtil.java:210)	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)	... 1 moreCaused by: org.apache.solr.common.SolrException: Error loading class &apos;solr.SynonymExpandingExtendedDismaxQParserPlugin&apos;	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:491)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:422)	at org.apache.solr.core.SolrCore.createInstance(SolrCore.java:567)	at org.apache.solr.core.PluginBag.createPlugin(PluginBag.java:122)	at org.apache.solr.core.PluginBag.init(PluginBag.java:217)	at org.apache.solr.core.PluginBag.init(PluginBag.java:206)	at org.apache.solr.core.SolrCore.<init>(SolrCore.java:764)	... 9 moreCaused by: java.lang.ClassNotFoundException: solr.SynonymExpandingExtendedDismaxQParserPlugin	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)	at java.security.AccessController.doPrivileged(Native Method)	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)	at java.net.FactoryURLClassLoader.loadClass(URLClassLoader.java:793)	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)	at java.lang.Class.forName0(Native Method)	at java.lang.Class.forName(Class.java:278)	at org.apache.solr.core.SolrResourceLoader.findClass(SolrResourceLoader.java:475)	... 15 more,code=500}</pre></p><hr><i><small>Powered by Jetty://</small></i><hr/></body></html>
2016-05-11 14:37:57 INFO     core.utils:45 add_data_set_to_neo4j() - Add dataset (uuid: 499b08ff-575e-4a8e-bb10-0bf5063de79e) to Neo4J and give read access to user (id: 4)
```
### Expected behavior
No error logged or displayed.

**Comment by hackdna** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;0dqHQzfq&#x2F;34-deploy-scc-release-on-aws"><img src="https:&#x2F;&#x2F;github.trello.services&#x2F;images&#x2F;trello-icon.png" width="12" height="12"> deploy SCC release on AWS</a>

**Comment by hackdna** [comment]
Potentially due to `SOLR_LIB_DIR = "/vagrant/refinery/solr/lib"` hardcoded in `refinery/config/settings/base.py`.

**Comment by ngehlenborg** [comment]
Related to #1153.

**Comment by jkmarx** [comment]
@drj11 Should we move this issue to the Yarmouth milestone?

**Comment by drj11** [comment]
Where is the celery log? @hackdna

**Comment by scottx611x** [comment]
@drj11 `refinery/log/celeryd-w1.log` & `refinery/log/celeryd-w2.log`

**Comment by drj11** [comment]
Fixed by 9ed6eeb76552bae05e5521705d187b3fd3de6af1 and 3279f8ebdaf5a735f73b3c083a3ae77b003dcbfb

Now when I upload `rfc-test.zip` the `refinery/log/celeryd-w1.log` shows:

```
2016-06-15 09:28:28 INFO     core.search_indexes:113 prepare() - Start preparing 'Test 1: Request for Comments (RFC) Test' for indexing
2016-06-15 09:28:28 INFO     core.search_indexes:145 prepare() - Successfully prepared 'Test 1: Request for Comments (RFC) Test' for indexing
2016-06-15 09:28:29 INFO     core.utils:48 add_data_set_to_neo4j() - Add dataset (uuid: 1c260710-2e8c-40f0-aae0-c5460b951b04) to Neo4J and give read access to user (id: 3)
```

**Comment by hackdna** [comment]
I've added log file info here: https://github.com/parklab/refinery-platform/wiki/operations#logs
